docs/src/pages/blog/everything.md

```diff
@@ -0,0 +1,163 @@
+
+So what do you get with `Patient/$everything`, well, following the HL7 FHIR R4 [specification](http://hl7.org/fhir/patient-operation-everything.html), you get an operation that returns a "searchset" bundle with the information related to a patient in the context it is invoked. In the IBM FHIR Server's case, the server returns whatever resources are related to the Patient - e.g. practitioners, medications, locations, organizations etc.  You also get support for `Patient/[id]/$everything` where the `[ID]` is the logical id specific to a patient.
+
+Let's build a request and see it in action:
+
+We need to know about the parameters of the `$everything` operation. 
+
+| Name | Type | Description |
+|---|---|---|
+| `start` | date | The date range relates to care dates, not record currency dates - e.g. all records relating to care provided in a certain date range. If no start date is provided, all records prior to the end date are in scope. |
+| `end` | date | The date range relates to care dates, not record currency dates - e.g. all records relating to care provided in a certain date range. If no end date is provided, all records subsequent to the start date are in scope. |
+| `_since` | date | Resources updated after this period will be included in the response. The intent of this parameter is to allow a client to request only records that have changed since the last request, based on either the return header time, or or (for asynchronous use), the transaction time|
+| `_type` | code | One or more parameters, each containing one or more comma-delimited FHIR resource types to include in the return resources. In the absence of any specified types, the server returns all resource types | 
+| `return` | Bundle | The bundle is "searchset" |
+
+Note, currently the implementation ignores the `_count` parameter.
+
+Now, let's prepopulate some sample data on the server using the [Integration Test data](https://github.com/IBM/FHIR/blob/main/fhir-server-test/src/test/resources/testdata/everything-operation/Antonia30_Acosta403.json).
+
+1. Download the Sample Data
+
+``` sh
+curl -L https://raw.githubusercontent.com/IBM/FHIR/main/fhir-server-test/src/test/resources/testdata/everything-operation/Antonia30_Acosta403.json -o Antonia30_Acosta403.json
+```
+
+2. Load the Sample Data bundle to the IBM FHIR Server
+
+``` sh
+curl -k --location --request POST 'https://localhost:9443/fhir-server/api/v4' \
+--header 'Content-Type: application/fhir+json' \
+--header "Authorization: Basic ${DUMMY_PASSWORD}" \
```

Comment:
should we steer them toward the `-u` option instead?  its a little easier than getting the base64 just right

Reply:
good point - fixed